core-web/libs/dotcms-webcomponents/.storybook/main.js

```diff
@@ -0,0 +1,29 @@
+const rootMain = require('../../../.storybook/main');
+
+
+
+module.exports = {
```

Comment:
Can we add a comment with a link to the documentation where we found this configuration? Something like:

// Find the configuration here: https://storybook.js.org/